src/solvers/flattening/boolbv.cpp

```diff
@@ -148,6 +151,8 @@ boolbvt::convert_bv(const exprt &expr, optionalt<std::size_t> expected_width)
   return cache_result.first->second;
 }
 
+/// Prints that the expression of {x} has failed conversion,
+/// then returns a vector of x's width with empty values.
```

Comment:
What are empty values in the context of literals? Anyway, I don't think it's necessary to specify the content of the returned literals here.